bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/internal/form/FormConstants.java

```diff
@@ -142,4 +142,8 @@ private FormConstants() {
 
     /* The resource type for the pre-selected the linked panel */
     public final static String RT_FD_FORM_REVIEW_DATASOURCE_V1 = RT_FD_FORM_PREFIX + "review/v1/datasource";
+
+    public static final String X_ADOBE_FORM_DEFINITION = "x-adobe-form-definition";
```

Comment:
please add comments / java doc on why this constant is required.

Reply:
done